inbm/dispatcher-agent/dispatcher/xmlhandler.py

```diff
@@ -148,7 +148,7 @@ def get_attribute(self, xpath: str, attribute_name: str) -> str:
 
         @param xpath: path to key
         @param attribute_name: name of attribute
-        @return: attribute str if found else None
+        @return: value if found else None
```

Comment:
**Suggestion:** The get_attribute method's docstring incorrectly states that it returns 'None' if the value is not found. However, the method raises an XmlException instead. Update the docstring to reflect the actual behavior. [documentation]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        @return: value of the attribute if found
        @raises: XmlException if the attribute or element is not found
```